01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,18 @@
+let Number = 1
+
+function FizzBuzz (MaxNumber) {
```

Comment:
```suggestion
function fizzBuzz(maxNumber) {
```

のようにメソッド名も引数名も小文字で始めるようにしましょう。